pkg/services/provisioning/dashboards/file_reader.go

```diff
@@ -25,10 +25,12 @@ var (
 )
 
 type fileReader struct {
-	Cfg              *DashboardsAsConfig
-	Path             string
-	log              log.Logger
-	dashboardService dashboards.DashboardProvisioningService
+	Cfg                          *DashboardsAsConfig
+	Path                         string
+	log                          log.Logger
+	dashboardProvisioningService dashboards.DashboardProvisioningService
+	dashboardService             dashboards.DashboardService
```

Comment:
Need both but even though it is one struct seemed easier this way than doing casting to different interfaces each time, not sure.